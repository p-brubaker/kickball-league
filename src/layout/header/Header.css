.header {
    display: flex;
    justify-content: space-around;
    border-bottom: 3px solid green;
}
