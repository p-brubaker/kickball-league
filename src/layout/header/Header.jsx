import { Link } from 'react-router-dom'
import './Header.css'

export default function Header() {
    return (
        <div className="header">
            <Link to="/">Home</Link>
            <Link to="/players/view">Players</Link>
            <Link to="/teams/view">Teams</Link>
        </div>
    )
}
