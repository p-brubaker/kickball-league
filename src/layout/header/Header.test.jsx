import { render } from '@testing-library/react'
import { MemoryRouter } from 'react-router-dom'
import Header from './Header.jsx'

it('should render the header with links to home, players, teams', async () => {
    const { container } = render(
        <MemoryRouter>
            <Header />
        </MemoryRouter>
    )

    expect(container).toMatchSnapshot()
})
