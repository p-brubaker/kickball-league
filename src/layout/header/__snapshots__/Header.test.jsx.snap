// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the header with links to home, players, teams 1`] = `
<div>
  <div
    class="header"
  >
    <a
      href="/"
    >
      Home
    </a>
    <a
      href="/players/view"
    >
      Players
    </a>
    <a
      href="/teams/view"
    >
      Teams
    </a>
  </div>
</div>
`;
